packages/arb-token-bridge-ui/src/util/orbitChainsList.ts

```diff
@@ -0,0 +1,113 @@
+import { ETHER_TOKEN_LOGO } from '../constants'
+import { ChainWithRpcUrl } from './networks'
+
+type BridgeConfigUi = {
+  primaryColor: `#${string}`
+  secondaryColor: `#${string}`
+  networkName: string
+  networkLogo: string
+  nativeTokenLogo: string
```

Comment:
I think we can make both of these optional, and have `networkLogo` default to Arbitrum Orbit, and `nativeTokenLogo` to ETH logo

Reply:
as discussed, only made `nativeTokenLogo` optional